packages/framework/src/Model/Cart/Cart.php

```diff
@@ -151,4 +153,37 @@ protected function findSimilarCartItemByCartItem(CartItem $cartItem)
 
         return null;
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Customer\CustomerIdentifier $customerIdentifier
+     * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
+     * @param int $quantity
+     * @param \Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceCalculationForUser $productPriceCalculation
+     * @param \Shopsys\FrameworkBundle\Model\Cart\Item\CartItemFactoryInterface $cartItemFactory
+     * @return \Shopsys\FrameworkBundle\Model\Cart\AddProductResult
+     */
+    public function addProductToCart(
```

Comment:
maybe simply `Cart::addProduct()` would be a better name? :slightly_smiling_face: